product/views.py

```diff
@@ -1,3 +1,58 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+from django.db.models import Prefetch
+from user.models import Creator,User
+from request.models import CourseRequest
+from product.models import (
+    Course,
+    CourseStatus,
+    Category,
+    SubCategory,
+    ProductImage,
+    Product,
+    ProductLike,
+)
+
+#클래스 좋아요 기능
+class CourseLikeView(View):
```

Comment:
정확히는 ProductLike 객체 관련된 코드이므로 이름을 수정해주는게 좋을 것 같습니다.
```suggestion
class ProductLikeView(View):
```

Reply:
수정완료했습니다